plugins/outputs/stackdriver/stackdriver.go

```diff
@@ -156,11 +159,19 @@ func (s *Stackdriver) Write(metrics []telegraf.Metric) error {
 
 			metricKind, err := getStackdriverMetricKind(m.Type())
 			if err != nil {
-				s.Log.Errorf("Get metric failed: %s", err)
+				s.Log.Errorf("Get kind for metric '%s' type '%T' field '%s' failed: %s", m.Name(), m.Type(), f, err)
```

Comment:
Please use the `%q` formatter which quotes strings, maybe check throughout the code
```suggestion
				s.Log.Errorf("Get kind for metric %q (%T) field %q failed: %s", m.Name(), m.Type(), f, err)
```

Reply:
Reasonable; do you have an opinion about the case of the variable being at end of string after a colon, e.g. on L243?

Reply:
For line 243, that's OK IMO. I usually use `%q` to highlight that piece or (and more importantly) to see if there are leading or trailing spaces etc... I'm also fine with `%s` just not with `'%s'` as this is what `%q` is for. :-)

Reply:
👍 okay, then I think we're good here.